schedule/yast/msdos.yaml

```diff
@@ -0,0 +1,73 @@
+---
+name:           msdos@yast
+description:    >
+  Test for installation on msdos partition table.
+schedule:
+  - installation/bootloader_start
+  - installation/welcome
+  - installation/accept_license
+  - installation/scc_registration
+  - installation/addon_products_sle
+  - installation/system_role
+  - installation/partitioning
+  - installation/partitioning/msdos_partition_table
+  - installation/installer_timezone
+  - installation/user_settings
+  - installation/user_settings_root
+  - installation/resolve_dependency_issues
+  - installation/installation_overview
+  - {{disable_grub_timeout}}
```

Comment:
I would call it more generic, that conditional blocks is not only for `disable_grub_timeout`, something like `installation_process` ?